public/partials/admin-apps-activities.html

```diff
@@ -0,0 +1,51 @@
+<div ng-include src="'/partials/common/navigation.html'"></div>
```

Comment:
These two navigation files should be a part of the main ejs page.  No need for them to be continuously changes for every page that is loaded.  @thinkking